packages/extension/webpack.config.js

```diff
@@ -18,109 +18,92 @@ const packages = [
+              limit: 10000,
+              name: 'static/[name].[ext]'
             }
-          ]
-        }
-      ]
-    },
-    node: {
-      child_process: 'empty',
-      dgram: 'empty',
-      fs: 'empty',
-      net: 'empty',
-      tls: 'empty'
-    },
-    optimization: {
-      concatenateModules: false,
-      moduleIds: 'natural',
-      occurrenceOrder: false,
-      providedExports: false,
-      sideEffects: false,
-      usedExports: false
-    },
-    output: {
-      chunkFilename: '[name].js',
-      filename: '[name].js',
-      globalObject: '(typeof self !== \'undefined\' ? self : this)',
-      path: path.join(context, 'build')
-    },
-    performance: {
-      hints: false
-    },
-    plugins: [
-      new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
-      new webpack.DefinePlugin({
-        'process.env': {
-          NODE_ENV: JSON.stringify(ENV),
-          PKG_NAME: JSON.stringify(pkgJson.name),
-          PKG_VERSION: JSON.stringify(pkgJson.version)
-        }
-      }),
-      new CopyPlugin({ patterns: [{ from: 'public' }] }),
-      new ManifestPlugin({
-        config: {
-          base: manifest,
-          extend: {
-            version: pkgJson.version.split('-')[0] // remove possible -beta.xx
           }
+        ]
+      }
+    ]
+  },
+  output: {
+    chunkFilename: '[name].js',
+    filename: '[name].js',
+    globalObject: '(typeof self !== \'undefined\' ? self : this)',
+    path: path.join(__dirname, 'build')
+  },
+  performance: {
+    hints: false
+  },
+  plugins: [
+    new webpack.ProvidePlugin({
+      Buffer: ['buffer', 'Buffer'],
+      process: 'process/browser'
+    }),
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('production'),
+        PKG_NAME: JSON.stringify(pkgJson.name),
+        PKG_VERSION: JSON.stringify(pkgJson.version)
+      }
+    }),
+    new CopyPlugin({ patterns: [{ from: 'public' }] }),
+    new ManifestPlugin({
+      config: {
+        base: manifest,
+        extend: {
+          version: pkgJson.version.split('-')[0] // remove possible -beta.xx
         }
-      })
-    ].filter((entry) => entry),
-    resolve: {
-      alias,
-      extensions: ['.js', '.jsx', '.ts', '.tsx']
-    },
-    watch: !isProd
-  };
-}
-
-module.exports = createWebpack({
-  alias: packages.reduce((alias, pkg) => {
-    alias[`@polkadot/${pkg}`] = path.resolve(__dirname, `../${pkg}/src`);
+      }
+    })
+  ].filter((entry) => entry),
+  resolve: {
+    alias: packages.reduce((alias, pkg) => {
+      alias[`@polkadot/${pkg}`] = path.resolve(__dirname, `../${pkg}/src`);
 
-    return alias;
-  }, {}),
-  context: __dirname
-});
+      return alias;
+    }, {}),
+    extensions: ['.js', '.jsx', '.ts', '.tsx'],
+    fallback: {
+      crypto: require.resolve('crypto-browserify'),
+      path: require.resolve('path-browserify'),
+      stream: require.resolve('stream-browserify')
```

Comment:
Adding these packages is what made the PR so big, `yarn.lock` being a bit more bloated.